src/Tribe/PUE/Checker.php

```diff
@@ -397,7 +397,7 @@ public function ajax_validate_key() {
 
 				// For multisite, return the network-level siteurl ... in
 				// all other cases return the actual URL being serviced
-				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : $_SERVER['SERVER_NAME'];
+				$queryArgs['domain'] = is_multisite() ? $this->get_network_domain() : ( defined( 'WP_SITEURL' ) ? parse_url( WP_SITEURL )['host'] : $_SERVER['SERVER_NAME'] );
```

Comment:
Hi Sun, while it is true WordPress recommends 5.6+, but they support  5.2.4+.
In fact in the past when we sneak in 5.3+ coding in there we generate a lot of support from people still running 5.2, which is a giant headache and takes away from us being able to improve the plugin. So we are going to continue supporting it until WordPress finally moves from it. We hope that they drop it and move forward, but they have to lead the charge unfortunately.